docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -281,15 +281,15 @@ async def main():
     arm = ArmClient.from_robot(robot=robot, name='myArm')
     await randomMovement(arm)
 
-    await robot.close()
+    await machine.close()
```

Comment:
this you cannot do unless you also change 276

Reply:
```suggestion
    await robot.close()
```
ooh yeah that was an accident thanks for catching